data/hai/hai reveal 4 middle.txt

```diff
@@ -1462,34 +1474,425 @@ mission "Hai Reveal [A11] Sound the Alarm"
+					goto human
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: human book"
+				`	HUM-9XC401Q: The Pirate Problem`
+					goto pirates
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: pirates book"
+				`	UHI-001AX94: Unfettered Military Activity and its Economic Impact on the Hai`
+					goto unfettered
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: unfettered book"
+				`	UHI-00201A3: Unfettered Food Situation`
+					goto acorns
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: acorns book"
+				`	CHS-015TNP3: A Lifetime of Disruption: Teeneep Before Public Life`
+					goto teeneep1
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: teeneep1 book"
+				`	QRG-3Z-18FJ4KC: Modern Hai-Quarg Relations`
+					goto quarg
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: quarg book"
+				`	"I've changed my mind. Take me back to the proceedings instead."`
+					goto reading
+					to display
+						"Hai Reveal [A11] Sound the Alarm: time" == 2
+				`	"I'm taking a nap. Let me know when it's all over."`
+					goto nap
+					to display
+						"Hai Reveal [A11] Sound the Alarm: time" > 2
+
+			label solutions
+			action
+				set "Hai Reveal [A11] Sound the Alarm: solutions book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+				log `A Hai intelligence report identifies three problems: "northern troubles," degrading secrecy of the Hai, and piracy. They expect piracy to vanish from the Republic "befriending" anarchist colonies and suggest Hai slowly reveal themselves through leaked information. It is unclear if that ever happened. For the "northern troubles," they have a "skilled mercenary" in mind for the job, and a diplomat.`
+			`	This one-page report is recent, and includes references to other reports you can access.`
+			# The "northern troubles" refers to the Unfettered raids on the Wanderers (who are to the north) not the troubles at Wah Ki (which is to the east). The Elders know, at some level, of the Unfettered-Wanderer hostilities. They're hoping to solve it through diplomacy (such as Sayari, INT-8C-131X04) or a skilled mercenary (such as Captain Last, INT-7Z-G81X51). In this mission, they gently nudge Captain Last to the north. They're using the captain's curious nature to lure the Captain into the conflict.
+			`	"While we can all agree that the northern troubles could escalate to dangerous levels we've identified ways of improving relations through diplomacy [INT-8C-131X04] or a skilled mercenary [INT-7Z-G81X51]. It would appear that there are more immediate problems. Escalation of northern troubles is probably several decades away [UHI-001AX94], but total failure of the joint Hai-human disinformation campaign is expected in significantly less than that [HUM-9A38HF]. We must focus on the growing improbability of the secrecy of the Hai [HUM-931XHF9], and increasing pirate problems [HUM-9XC401Q]. Solving all three problems will open a path for Hai unification.`
```

Comment:
```suggestion
			`	"While we can all agree that the northern troubles could escalate to dangerous levels we've identified ways of improving relations through diplomacy [INT-8C-131X04] or a skilled mercenary [INT-7Z-G81X51]. It would appear that there are more immediate problems. Escalation of northern troubles is probably several decades away [UHI-001AX94], but total failure of the joint Hai-human disinformation campaign is expected in significantly less than that [HUM-9A38HF]. We must focus on the growing improbability of the secrecy of the Hai [HUM-931XHF9], and increasing pirate problems [HUM-9XC401Q]. Solving all three problems will open a path for Hai unification."`
```